src/common/models/Projection.ts

```diff
@@ -219,16 +223,28 @@ export class Projection {
       this.rawDailyDeaths,
     );
 
-    this.rawHospitalizations = actualTimeseries.map(
-      row => row && row.hospitalBeds.currentUsageCovid,
+    // Disaggregate county hospitalization data from HSAs to counties in order to display
+    // county estimates in charts.
+    this.rawHospitalizations = actualTimeseries.map(row =>
```

Comment:
This seems like a lot of ternary operators--I'm willing to split these calculations up if that's preferred